chipsec/module_common.py

```diff
@@ -36,8 +43,58 @@ class ModuleResult:
     INFORMATION = 5
     NOTAPPLICABLE = 6
     ERROR = -1
-
-
+    # -------------------------------------------------------
+    class status(Enum):
```

Comment:
What about inconsistent settings where only some of the bits are set correctly?

Reply:
There's lots of bits available still, we can have one or more for that case if needed. There'll probably be more added as tests are being modified to add support.

Reply:
Shouldn't that work be done first.  Going through the various modules to see potential areas that need to be highlighted so the list is comprehensive before editing them partially and retrofitting if more are added?

Reply:
It was done first, that's where that list came from. Hopefully nothing was missed but it's open for change/upgrade/improvement as modules are modified.